data/products.json

```diff
@@ -4,192 +4,529 @@
     "country_code": "US",
     "alternatives": [
       {
-        "name": "GIMP",
-        "country_code": "XX",
+        "name": "GIMP", 
+        "country_code": "CA", 
         "link": "https://www.gimp.org/"
-      },
-      {
-        "name": "Affinity Photo",
```

Comment:
Any reason for deleting 'Affinity Photo' alternative?

Consider same remark for others where we deleted some alternatives